main/datalog.go

```diff
@@ -216,16 +216,17 @@ func bulkInsert(tbl string, db *sql.DB) (res sql.Result, err error) {
 
 	batchVals := insertBatchIfs[tbl]
 	for len(batchVals) > 0 {
-		i := int(0) // Maximum of 25 rows per INSERT statement.
+		i := int(0)
 		stmt := ""
 		vals := make([]interface{}, 0)
-		querySize := uint64(0) // Size of the query in bytes.
-		for len(batchVals) > 0 && i < 1000 && querySize < 750000 {
+		querySize := uint64(0)                                   // Size of the query in bytes.
+		for len(batchVals) > 0 && i < 30 && querySize < 750000 { // Maximum of 1000? columns for INSERT on default sqlite compile. trafficData is limiting function with 31 parameters.
```

Comment:
Also, are you talking about rolling up ALL records from everyone? Makes 
sense to do that, just requires one extra field.



On 04/03/2016 01:17 PM, cyoung wrote:
>
> In main/datalog.go 
> <https://github.com/cyoung/stratux/pull/365#discussion_r58314775>:
>
> >  		stmt := ""
> >  		vals := make([]interface{}, 0)
> > -		querySize := uint64(0) // Size of the query in bytes.
> > -		for len(batchVals) > 0 && i < 1000 && querySize < 750000 {
> > +		querySize := uint64(0)                                   // Size of the query in bytes.
> > +		for len(batchVals) > 0 && i < 30 && querySize < 750000 { // Maximum of 1000? columns for INSERT on default sqlite compile. trafficData is limiting function with 31 parameters.
>
> @skypuppy <https://github.com/skypuppy> - how would this work?
> The idea behind this is to correlate all of the data that we have, let 
> it be properly timestamped, and have it work across the multiple 
> hardware profiles we have now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cyoung/stratux/pull/365/files/a546f18750e1ebf0f5d5086889ab3513ed0deb3a#r58314775>
>